README.adoc

```diff
@@ -81,30 +81,30 @@ You will update the methods in the template interface for this client to be asyn
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Replace the `JobGatewayClient` interface.#
-`gateway/src/main/java/io/openliberty/guides/gateway/client/JobGatewayClient.java`
+#Replace the `JobClient` interface.#
+`gateway/src/main/java/io/openliberty/guides/gateway/client/JobClient.java`
 ----
-JobGatewayClient.java
+JobClient.java
 [source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/gateway/src/main/java/io/openliberty/guides/gateway/client/JobGatewayClient.java[]
+include::finish/gateway/src/main/java/io/openliberty/guides/gateway/client/JobClient.java[]
 ----
 
 The changes involve changing the return types of the [hotspot=getJobs file=0]`getJobs`, [hotspot=getJob file=0]`getJob`, and [hotspot=createJob file=0]`createJob` methods to return `CompletionStage<T>` types. These changes make the methods asynchronous. Since the methods now have return type of `CompletionStage<T>`, you aren't able to directly manipulate the inner types. As you will see in the next section, you will be able to indirectly use the inner object by chaining callbacks.
 
-== Responding asynchronously to an HTTP request
+== Updating REST resource to handle HTTP requests asynchronously
```

Comment:
Is just one REST resource being updated in this example? If so, my suggested update is: 
"Updating the REST resource to asynchronously handle HTTP requests"

Reply:
In this section, we are only updating one REST resource.  The service itself has multiple resources.  I think having "the" is fine here?  Otherwise, we will have to do something like "Updating REST resource for gateway service ...", which is longer.

Reply:
Once we close on this, @jonathanxue, we can merge.

Reply:
@yeekangc How about "a" instead of "the"? I think it reads better with an article in front of REST resource. So it would be:
"Updating a REST resource to asynchronously handle HTTP requests"

Reply:
I went back and forth between "a" and "the" so I am good with it too.  =)